src/main/java/com/gmail/woosay333/onlinebookstore/controller/BookController.java

```diff
@@ -19,30 +22,35 @@
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-@RequestMapping("/api/books")
+@RequestMapping("/books")
 @RequiredArgsConstructor
+@Tag(name = "Book management", description = "Endpoints for managing books")
 public class BookController {
     private final BookService bookService;
 
     @GetMapping
-    public List<BookDto> getAllBooks() {
-        return bookService.findAll();
+    @Operation(summary = "Get all books", description = "Get all books")
```

Comment:
not sure it makes sense to just duplicate the summary in the description, might as well leave it blank, but we can instead add more details, like that it supports pagination and sorting for example